client/src/data/testimonials.js

```diff
@@ -1,44 +1,41 @@
 export default [
   {
-    person: 'Kendalyn Hesterson',
-    picture:
-      'https://media.licdn.com/dms/image/C5603AQE4fiq2wYLN4w/' +
-      'profile-displayphoto-shrink_200_200/0?e=1575504000&v=beta&' +
-      't=ENWJca_rj3uyCKSjckj0IkVQj-Z__6hCr9-rSIDvQOc',
-    country: 'Switzerland',
-    testimonial: `Lorem ipsum dolor amet mustache knausgaard +1, 
-            blue bottle waistcoat tbh semiotics artisan synth stumptown 
-            gastropub cornhole celiac swag. Brunch raclette vexillologist 
-            post-ironic glossier ennui XOXO mlkshk godard pour-over blog 
-            tumblr humblebrag. Blue bottle put a bird on it twee prism 
-            biodiesel brooklyn. Blue bottle ennui tbh succulents.`,
+    person: 'Daniella Di Stefano',
+    picture: '/asset/images/daniella_stefano.JPG',
+    // 'profile-displayphoto-shrink_200_200/0?e=1575504000&v=beta&' +
+    // 't=ENWJca_rj3uyCKSjckj0IkVQj-Z__6hCr9-rSIDvQOc',
```

Comment:
We can probably just delete these lines in all of the entries

Reply:
Yup, lets go ahead and delete all the commented stuff :)